tools/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# SPDX-FileCopyrightText: 2023 Volker Krause <vkrause@kde.org>
+# SPDX-License-Identifier: MIT
+
+scriptdir=`dirname "${BASH_SOURCE[0]:-$0}"`
+datadir=$scriptdir/../data
+
+
+
+for j in `find $datadir -name "*.json"`; do
+    country=`echo "$j" | sed -e 's,^.*/\([a-z][a-z]\)/.*$,\1,'`
```

Comment:
This seems very brittle.

Reply:
It is. The alternative I was considering is an explicit mapping to the bounding box for each file that needs one, much more flexible but also more maintenance/duplication. Not pretty either.